package.json

```diff
@@ -44,16 +44,16 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.4.3",
-    "@carbon/colors": "10.7.2",
-    "@carbon/grid": "10.8.2",
-    "@carbon/icons-react": "10.8.2",
+    "@carbon/colors": "10.8.0-rc.0",
+    "@carbon/grid": "10.9.0-rc.0",
+    "@carbon/icons-react": "10.9.0-rc.0",
     "@carbon/import-once": "10.3.0",
-    "@carbon/layout": "10.7.3",
-    "@carbon/motion": "10.5.2",
-    "@carbon/themes": "10.9.3",
-    "@carbon/type": "10.8.3",
-    "carbon-components": "10.9.3",
-    "carbon-components-react": "7.9.3",
+    "@carbon/layout": "10.8.0-rc.0",
+    "@carbon/motion": "10.6.0-rc.0",
+    "@carbon/themes": "10.10.0-rc.0",
+    "@carbon/type": "10.9.0-rc.0",
+    "carbon-components": "10.10.0-rc.0",
+    "carbon-components-react": "7.10.0-rc.0",
```

Comment:
Because of these updates, a number of snapshots need to be regenerated. 🙃 
But to keep the subset of changes here low, I'd like to avoid updating snapshots in this branch right now. Let me know if you have concerns about this though. 👍

Reply:
Good idea! Let's update these towards the end to mitigate noise